app/javascript/components/patient/close_contact/CloseContact.js

```diff
@@ -174,12 +179,12 @@ class CloseContact extends React.Component {
       });
   };
 
-  createModal(title, toggle, handleCloseContactSubmit) {
+  createModal() {
     return (
-      <Modal size="lg" show centered onHide={toggle}>
-        <h1 className="sr-only">{title}</h1>
+      <Modal size="lg" show centered onHide={this.toggleCloseContactModal}>
+        <h1 className="sr-only">{this.props.close_contact.id ? 'Update Close Contact' : 'Create Close Contact'}</h1>
```

Comment:
You can just do this: {this.props.close_contact.id ? 'Update' : 'Create'} Close Contact

Reply:
still needs to be done